addons/website/static/src/snippets/s_timeline/001.scss

```diff
@@ -0,0 +1,34 @@
+.s_timeline[data-vcss="001"] {
+    .s_timeline_dot {
+        width: var(--o-timeline-dot-size, 24px);
+        height: var(--o-timeline-dot-size, 24px);
```

Comment:
> Question: why did you need to define a variable, given that nothing uses it or sets it (it is undefined) ?

It was for good practice :) This way, the "pro users" can easily change the value if they want to edit the design of this element.

> I would simply declare --o-timeline-dot-size: 24px above (just under .s_timeline) so it is always defined and use var(--o-timeline-dot-size) (without the 24px fallback, since it is always defined) instead 

Indeed, but using a fallback value (`24px`) ensures that the dots will still work if the variable isn't loaded for some reason (for example, if it's been accidentally deleted 😬 ).
This way, we'll only define the variable if we want to customize the value, which is cleaner IMO.

What do you think ? 🙂

Reply:
Okay then , I guess it is okay like this :slightly_smiling_face: 

Just IMO, I think we should not really think about the "pro users" when making things, since the goal of the editor is to make things easy for everyone, which is why there are options to customize styles like that :thinking:  After all, the pro user could also simply add rules in CSS setting the width and height => the variable itself is not really needed (except for repeating the value at several places) :+1: 
So here, if you thought that a "pro user" might want to customize the variable, then that means that you think some users will want to change the size of the dot, which is why adding an option automatically changing the value could have been a good idea :slightly_smiling_face: 
Just to say that I think it is better to think in terms of "options for everyone", instead of "easy to access/customize variables for a few users" :wink: 

For now, since the specs were not mentioning this options, I guess we can go without it :+1: (Maybe ask the PO for his opinion ? :slightly_frowning_face:)